docs/05-community/01-contribution-guide.mdx

```diff
@@ -339,7 +339,7 @@ related:
 
 ### Nested Fields
 
-| Field         | Required? | Description                                                                                                                                               |
+| Field         | Requirement | Description                                                                                                                                               |
```

Comment:
here `Required` makes more sense than `Requirement`

Reply:
In the sense of grammar used here, the word `Requirement` fits more appropriately than `Required` because the following values **'Optional'** and **'Required'** do not meet the perfect subject-verb agreement. `Requirement` feels easier to grasp for the readers.

Also, a value should not be the same as the heading of a column.
This is a minimal change fix for the same.

Reply:
@Ishaan2053 Sorry I forgot to comment here too. I think this is fine to leave it as it was since most of our Docs just use the word `Required`.

![CleanShot 2024-01-17 at 17 14 33@2x](https://github.com/vercel/next.js/assets/28912696/596cc3e0-5a16-4047-beb3-05c09c0fa3fc)

```suggestion
| Field         | Required? | Description                                                                                                                                               |
```

Reply:
No issues! And I guess you're right, it keeps the consistency in language used.